purchase_line_product_image/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please can you add headers to all files following [OCA's template module](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1-L3)

Reply:
yes, it's done